kernel/aster-nix/src/thread/work_queue/mod.rs

```diff
@@ -141,6 +141,9 @@ impl WorkQueue {
             .lock_irq_disabled()
             .pending_work_items
             .push(work_item);
+        if let Some(worker_pool) = self.worker_pool.upgrade() {
+            worker_pool.schedule()
```

Comment:
Now that the monitor task can run at a high priority, do you think calling `worker_pool.schedule` is necessary? I am worried that doing so adds relatively high overhead to the `enqueue` method.

Reply:
I think we can check the priority of `WorkerPool`. If the priority is high then we can call this method to handle the work item in time.